client/modules/Booking/Booking.js

```diff
@@ -57,92 +115,117 @@ export class Booking extends Component {
-            <form onSubmit={this.handleSubmit}>
-                <label className={style.labels} htmlFor="location">
-                    <div className={style.labelText}>Location *</div>
-                    <select name="location">
-						<option value="location1">Location 1</option>
-						<option value="location2">Location 2</option>
-						<option value="location3">Location 3</option>
-						<option value="location4">Location 4</option>
-					</select>
-                </label>
-				<br/>
-                <label className={style.labels} htmlFor="Car">
-                    <div className={style.labelText}>Car *</div>
-                    <select name="Car">
-						<option value="car1">Car 1</option>
-						<option value="car2">Car 2</option>
-						<option value="car3">Car 3</option>
-						<option value="car4">Car 4</option>
-					</select>
-                </label>
-				<br/>
-                <label className={style.labels} htmlFor="startDate">
-                    <div className={style.labelText}>Start Date *</div>
-                    <input type="date" name="startDate" id="startDate"/>
-                </label>
-				<p className={style.note}>Please provide date in mm/dd/yyyy</p>
-				<br/>
-				<label className={style.labels} htmlFor="startTime">
-                    <div className={style.labelText}>Start Time *</div>
-                    <input type="time" name="startTime" id="startTime"/>
-                </label>
-				<p className={style.note}>Please provide time in hh:mm AM/PM</p>
-				<br/>
-				<label className={style.labels} htmlFor="endDate">
-                    <div className={style.labelText}>End Date *</div>
-                    <input type="date" name="endDate" id="endDate"/>
-                </label>
-				<p className={style.note}>Please provide date in mm/dd/yyyy</p>
-				<br/>
-				<label className={style.labels} htmlFor="endTime">
-                    <div className={style.labelText}>End Time *</div>
-                    <input type="time" name="endTime" id="endTime"/>
-                </label>
-				<p className={style.note}>Please provide time in hh:mm AM/PM</p>
+	  return (
+	  <div className={styles.body}>
+		<Container>
+		<Row>
+			<Col>
+	        <h1 className={styles.title}>Booking</h1>
+			</Col>
+		</Row>
+            <form onSubmit={this.handleSubmit.bind(this)}>
+        <Row>
+			<Col xs="12" sm="6">
+			<hr/>
+                <h4 className={styles.h4}>Check availability for:</h4>
+                <p>Vehicle - </p>
+                <p>Hire Type - </p>
+                <p>Registration - </p>
 				<br/>
-                <input type="submit" value="Submit" />
+                <p>Address Placeholder</p>
+			</Col>
+			<Col xs="12" sm="6">
+			<hr/>
+			<h4 className={styles.h4}>Booking Form</h4>
+				<FormGroup>
+          {this.renderLabel("startDate", "startDate")}
+						<DatePicker 
+						className={this.isError('startDate') ? 'is-invalid' : ''}
```

Comment:
can we prefix className with form-control? ie {'form-control ' + this.isError('startDate') ? 'is-invalid' : ''}
it will make it look more bootstrappy